RELEASES.md

```diff
@@ -86,3 +86,79 @@ For that we need to take some precautions:
 Trying to make prereleases easier to handle we created a script that you can use on your branch before merging into master, that will bump the package you select, and you can choose if you it's an alpha or beta release, before publishing it to NPM.
 
 You can check the script [here](https://github.com/contentful/forma-36/blob/c6b10071959a085b21e49f5411a5ebff2f8a70d6/scripts/prerelease.mjs)
+
+## Next branch
+
+The next branch is used for the new version of Forma 36, which will probably have breaking changes. To help the development of the new packages, we have setup an automatic publishing of the packages on that branch to the `alpha` tag.
+These packages aren't shown on the website as they can have breaking changes often.
+
+### How to automatic publish
+
+To make changeset publish a new version of the packages it's the same as in the `main` branch, it just requires having a changeset generated and specifing the packages.
```

Comment:
❓ will this also be the `npx changeset `command? Maybe would be good to add the exact command then also here in this seciton

Reply:
Yeah, that is the same command, I'll add the command on the documentation to make it clear